test/tools/api.js

```diff
@@ -80,38 +80,42 @@ const asyncApi = [
   { className: 'Collection', method: 'mapReduce', returnType: 'Promise<Document | Document[]>' },
   { className: 'Collection', method: 'options', returnType: 'Promise<Document>' },
   { className: 'Collection', method: 'remove', returnType: 'Promise<DeleteResult> | void' },
-  { className: 'Collection', method: 'rename', returnType: 'Promise<Collection>' },
+  { className: 'Collection', method: 'rename', returnType: 'Promise<Collection>', special: 'returns transformed collection' },
   { className: 'Collection', method: 'replaceOne', returnType: 'Promise<UpdateResult | Document>' },
   { className: 'Collection', method: 'stats', returnType: 'Promise<CollStats>' },
   { className: 'Collection', method: 'update', returnType: 'Promise<UpdateResult> | void' },
   { className: 'Collection', method: 'updateMany', returnType: 'Promise<UpdateResult | Document>' },
   { className: 'Collection', method: 'updateOne', returnType: 'Promise<UpdateResult>' },
+  { className: 'Collection', method: 'initializeOrderedBulkOp', returnType: 'OrderedBulkOperation', special: 'not a callback accepting API' },
+  { className: 'Collection', method: 'initializeUnorderedBulkOp', returnType: 'UnorderedBulkOperation', special: 'not a callback accepting API' },
```

Comment:
what is the purpose of the description in "special" here?

Reply:
removed